5.0/5.0.7/debian/Dockerfile

```diff
@@ -37,11 +37,13 @@ RUN buildDeps="curl sudo python-setuptools python-dev build-essential libssl-dev
  && sudo -u plone bin/buildout \
  && chown -R plone:plone /plone /data \
  && rm -rf /Plone* \
- && SUDO_FORCE_REMOVE=yes apt-get purge -y --auto-remove $buildDeps \
+ && ADDED_PACKAGES=`apt-mark showauto` \
```

Comment:
Oh I guess the name of the var is confusing, it is not about removing apt-mark, apt-mark show auto will check for 'third party' packages/dependencies  installed by the buildDeps and will remove those.

you can also do instead:

``SUDO_FORCE_REMOVE=yes apt-get remove --purge -y $buildDeps $(apt-mark showauto) \``